packages/address/src/AddressFormatter.ts

```diff
@@ -44,6 +44,19 @@ export default class AddressFormatter {
     return countries;
   }
 
+  async getZoneName(
+    countryCode: string,
+    zoneCode: string,
+  ): Promise<string> {
+    const country = await this.getCountry(countryCode);
+    const countryZone = country.zones.find(
+      (item) => item.code === zoneCode,
+    );
+    if (!countryZone?.name) return '';
```

Comment:
If no zone can be found at the given code, we should return `undefined` instead of an empty string, to make it easier for the consumer to switch on the type of it and ensure they're handling the case where this could be an invalid code.

Reply:
@ryanwilsonperkin 
Thank you for your feedback. I have made the suggested changes as per your instructions.
https://github.com/Shopify/quilt/pull/2662/commits/18e9932e0e3e8926b6aa832c21373cf59faa121c
https://github.com/Shopify/quilt/pull/2662/commits/9dfe2ec9b868676a1172612d39bb50e1ff04272f